__tests__/app.test.js

```diff
@@ -64,3 +63,43 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("GET:200 - Responds with an article relevant to the ID ,specified as a parameter", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length).toBeGreaterThan(0);
+        expect(body).toMatchObject([
+          {
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: expect.any(String),
+            votes: 100,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          },
+        ]);
+      });
+  });
+  test("GET:404 - Responds with an error message of author id invalid if the author Id does not exist", () => {
+    return request(app)
+      .get("/api/articles/99999")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Author id invalid");
```

Comment:
I think that a msg like 'not found' rather than 'invalid' would be more fitting.  The id is valid, it's just doesn't exist!

Reply:
Also it's an article_id, not author_id 👍